ci/kind/test-e2e-kind.sh

```diff
@@ -367,6 +367,15 @@ function run_test {
   EXTRA_ARGS="$vlan_args --external-server-ips $(docker inspect external-server -f '{{.NetworkSettings.Networks.kind.IPAddress}},{{.NetworkSettings.Networks.kind.GlobalIPv6Address}}')"
 
   go test -v -timeout=$timeout $RUN_OPT antrea.io/antrea/test/e2e $flow_visibility_args -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --skip-cases=$skiplist $coverage_args $EXTRA_ARGS
+
+  if $coverage; then
+    pushd $ANTREA_COV_DIR
+    for dir in */; do 
```

Comment:
are there subdirectories in `ANTREA_COV_DIR` / `GOCOVERDIR`? I thought it was a flat structure of meta and counters files.

Reply:
There is atleast one directory for each agent/controller, each containing meta and counter files. So it makes a cov.out file for each directory, then removes the directory.

There can be more than one directory for an agent/controller if a test reuses that agent/controller after a transfer has been done. Most probably this situation won't happen, but to be safe, I added code to make sure that the new files are stored in a new dir.

Reply:
That makes sense. I guess `GOCOVERDIR` as a flat structure inside a Pod, but when we copy the files out to `ANTREA_COV_DIR`, we use different directories